### PR TITLE
add libwebp to the pinning list

### PR DESCRIPTION
**Pull request by bgruening**
This will add libwebp to the pinning list. According to https://abi-laboratory.pro/tracker/timeline/libwebp this can be pinned to X.Y.*.

### CODE REVIEW

File: scripts/pin_the_slow_way.py [hunk 1]
```diff
@@ -62,6 +62,7 @@
           'libssh2': 'libssh2 1.8.*',  # 1.8.0
           'libsvm': 'libsvm 3.21|3.21.*',  # NA
           'libtiff': 'libtiff >=4.0.3,<4.0.8',  # 4.0.6
+          'libwebp': 'libwebp 0.5.*',  # 0.5.2 https://abi-laboratory.pro/tracker/timeline/libwebp/
```
Comment (isuruf) [inline scripts/pin_the_slow_way.py]: `0.5.2` -> `0.5.0` (defaults has 0.5.0 and the commented versions show which version defaults channel is using)

Comment (bgruening) [inline scripts/pin_the_slow_way.py]: Thanks @isuruf, I thought it is the conda-forge version.

**Comment by jakirkham** [comment]
Could you please add the ABI Laboratory link in the OP?

**Comment by bgruening** [comment]
I have no clue what a OP is :)

**Pull request merged**